pylint_odoo/checkers/format.py

```diff
@@ -41,13 +44,20 @@
 NO_IDENTIFIED = -1
 
 
-class FormatChecker(BaseTokenChecker):
-
-    # Auto call to `process_tokens` method
-    __implements__ = (ITokenChecker)
+class FormatChecker(PylintOdooTokenChecker):
 
     name = settings.CFG_SECTION
     msgs = ODOO_MSGS
+    odoo_check_versions = {
+        'no-utf8-coding-comment': {
+            'min_odoo_version': (4, 2),
+            'max_odoo_version': (10, 0),
+        },
+        'unnecessary-utf8-coding-comment': {
+            'min_odoo_version': (11, 0),
+            'max_odoo_version': (11, 0),
```

Comment:
For `12.0` apply this same check so you should define just `min_odoo_version` for this case.

Reply:
Done, I deleted the max_odoo_version and keep only the min_odoo_version